src/gsi/cplr_get_wrf_mass_ensperts.f90

```diff
@@ -1705,6 +1740,338 @@ subroutine general_read_wrf_mass2(this,filename,g_ps,g_u,g_v,g_tv,g_rh,g_cwmr,g_
   return
   end subroutine general_read_wrf_mass2
 
+  subroutine general_read_wrf_mass_dirZDA (this,filename,g_qr,g_qs,g_qg,g_qnr,g_w,mype)
```

Comment:
The variable, this, was identified as unused.  Please remove unused variable.

Reply:
We added 'this' to be used in this subroutine when calling subroutine 'fill_regional_2d' like general_read_wrf_mass2

https://github.com/NOAA-EMC/GSI/pull/142/files#diff-f4844e36d71565749a9ceb0a3aa45ace24257a80b593048fbc2b5585e0bbca8fR2040-R2056